README.md

```diff
@@ -178,13 +176,54 @@ var app = builder.Build();
 app.UseAuthentication();
 app.UseAuthorization();
 
+// [...] Map your endpoints
+```
+
+#### OpenAPI integration
+
+If you are using [Swashbuckle](https://learn.microsoft.com/en-us/aspnet/core/tutorials/web-api-help-pages-using-swagger) to document your API, the `[RequireClientCredentials]` attribute will automatically populate the security definitions and requirements in the OpenAPI specification. For minimal APIs, there is a corresponding `RequireClientCredentials()` method. 
+
+For example:
+
+```csharp
+// Controlled-based approach
+[HttpGet]
+[Route("weather")]
+[RequireClientCredentials("read")]
+public async Task<IActionResult> GetWeather()
+{ /* ... */ }
+
 // Minimal APIs
-app.MapGet("/hello-world", () => "Hello World!").RequireAuthorization("my-policy");
+app.MapGet("/weather", () => { /* ... */ }).RequireClientCredentials("read");
+```
 
-// Controller-style
-[Authorize("my-policy")]
-[HttpGet("hello-world")]
-public IActionResult HelloWorld() => this.Ok("Hello world");
+Will generate this:
+
+```yaml
+paths:
+  /weather:
+    get:
+      summary: 'Required scope: read.'
+      responses:
+        '200':
+          description: OK
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+      security:
+        - clientcredentials:
+            - target-entity:b108bbc9-538e-403b-9faf-e5cd874eb17f:read # Based on the provided JwtBearerOptions.Audience
```

Comment:
What would be generated in a non FusionsAuth scenario?

Reply:
We only support generation of FusionAuth-compatible scopes